package.json

```diff
@@ -12,7 +12,7 @@
     },
     "scripts": {
         "test": "npx qunit-testrunner",
-        "test:keepAlive": "npx qunit-testrunner --keepalive",
```

Comment:
Is the port change necessary?

Reply:
It's not, but more convenient if you rebuild and restart testing server - then you'll need just to refresh opened test